pallets/schemas/src/lib.rs

```diff
@@ -383,7 +395,7 @@ pub mod pallet {
 			<CurrentSchemaIdentifierMaximum<T>>::set(n);
 		}
 
-		/// Build the [`Schema`] and insert it into storage
+		/// Build the [`SchemaInfo`] and insert it into storage
```

Comment:
This description is not quite correct. We add _both_ the `SchemaInfo` and the `SchemaPayload` to storage; it's just now they're separate storage.